lib/driver.js

```diff
@@ -388,6 +388,18 @@ class XCUITestDriver extends BaseDriver {
     return !(CAP_NAMES_NO_XCODEBUILD_REQUIRED.some((x) => Boolean(this.opts[x])));
   }
 
+  /**
+   * Whether to skip the device check in a new session request.
+   * This is a workaround for network connected devices such as an Apple TV 4k real device.
+   * A situation that requires this capability may hve an issue to communicate with the device
+   * over appium-ios-device for example.
+   * Some methods that require the appium-ios-device may get issues.
+   * @returns {boolean}
+   */
+  skipDeviceCheck() {
```

Comment:
there is no need for a separate method. We could still check the opt value where necessary